packages/snap-controller/src/Autocomplete/AutocompleteController.ts

```diff
@@ -867,12 +909,14 @@ function getAutocompleteSchemaData({
 		q: store.search?.originalQuery?.string || store.search?.query?.string || '',
 		correctedQuery: store.search?.originalQuery?.string ? store.search?.query?.string : undefined,
 		...filters,
-		sort: [
-			{
-				field: store.sorting.current?.field,
-				dir: store.sorting.current?.direction as AutocompleteSchemaDataSortInnerDirEnum,
-			},
-		],
+		sort: store.sorting.options
+			.filter((sort) => sort.active)
+			?.map((sort) => {
+				return {
+					field: sort.field,
+					dir: sort.direction as AutocompleteSchemaDataSortInnerDirEnum,
+				};
+			}),
```

Comment:
Unfortunately, after looking into this further, it appears the active property is set on the sort even if it is not "applied". We will likely need to use the request parameters instead to find the "applied" sorts.

<img width="978" alt="image" src="https://github.com/user-attachments/assets/b59d72f0-cd2e-4d0c-83d7-9b656f7e969c" />